source/collections/collation-collection.txt

```diff
@@ -0,0 +1,64 @@
+.. _collation-collection:
+
+==================================
+Create a Collection with Collation
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Collation allows users to specify language-specific rules for string 
```

Comment:
[suggestion]

Consider using the second person here: "Collation allows you to specify..."

Reply:
Done.